dev/src/components/NavBar.tsx

```diff
@@ -1,49 +1,56 @@
-import React from "react"
-import { useAppState } from "../overmind"
-import { Link } from "react-router-dom"
-import NavBarFooter from "./navbar/NavBarFooter"
-import NavBarCourse from "./navbar/NavBarCourse"
-import { isEnrolled, isVisible } from "../Helpers"
-
+import React from "react";
+import { useActions, useAppState } from "../overmind";
+import { Link } from 'react-router-dom'
+import NavBarFooter from "./navbar/NavBarFooter";
+import NavBarCourse from "./navbar/NavBarCourse";
+import { Color, isEnrolled, isFavorite } from "../Helpers";
+import NavFavorites from "./NavFavorites";
+import { Status } from "../consts"
+import NavBarUser from "./navbar/NavBarUser";
 
 //TODO Review the NavBar behaviour.
 const NavBar = (): JSX.Element => {
     const state = useAppState()
+    const actions = useActions()
 
-    const visible = state.enrollments.filter(enrollment => isEnrolled(enrollment) && isVisible(enrollment))
-
-    const courses = visible.map((enrollment) => {
-        return <NavBarCourse key={enrollment.getId()} enrollment={enrollment} />
-    })
+    const onCourseClick  = () => {
+        actions.setActiveFavorite(!state.showFavorites)
+    }
 
     return (
-        <nav className="navigator">
-            <ul key="list" className="SidebarList">
-                <li key="logo" className="logo">
-                    <Link to="/">
-                        QuickFeed
-                    </Link>
-                </li>
-                {!state.isLoggedIn &&
-                    <li>
-                        <a href="/auth/github" style={{ textAlign: "center", paddingTop: "15px" }}>
-                            Sign in with <i className="fa fa-2x fa-github align-middle ml-2" id="github" />
+        <nav className="navbar navbar-expand-lg" style={{backgroundColor: "#222", color: "#d4d4d4"}} id="main" >
+            {!state.showFavorites &&
+            <a className="navbar-brand" style={{marginLeft: "30px", fontWeight: "bold"}}>
```

Comment:
Missing indentations here. Please use the ESLint plugin to automagically format the code.